### ISSUE TITLE
Wrong label for objects in visualizations  

### ISSUE THREAD

**Issue by ethan-iai** [allenai/interactron#4]
Excellent work! Thanks very much for releasing the codes!
I am trying to evaluate the Interaction with the following command: 
```shell
python evaluate.py --config=configs/interactron.yaml
```
But the visualizations seems weird

![pos= -0 25,0 91,0 25 _rot= 240deg](https://user-images.githubusercontent.com/62320020/197149466-13658598-4513-4510-9d66-f05430746402.jpg)

After checking `tlvis_classes` in `utils/constants.py`, I find hat the index of wrongly-marked label is next to the 'real' label:

<img width="427" alt="截屏2022-10-21 下午4 39 35" src="https://user-images.githubusercontent.com/62320020/197152788-b9a17c26-ac8c-42d9-8ba8-289aaa517d67.png">
<img width="319" alt="截屏2022-10-21 下午4 41 47" src="https://user-images.githubusercontent.com/62320020/197153176-14351731-fc14-4230-b15e-3715150d9afe.png">
 
In conclusion, directly using category to index its text in `tlvis_classes` leads to the wrong labelling. To address this issue, I add a dummy item to `tlvis_classes`, and the visualisations look fine later.

**Comment by ethan-iai** [comment]
Hope this may work (please rename constant.py.docx to constant.py for outlook not supporting sending Python scripts).

Best regards.

From: LI Jichang ***@***.***>
Date: Thursday, December 22, 2022 at 9:32 PM
To: allenai/interactron ***@***.***>
Cc: Ethan Ng ***@***.***>, Author ***@***.***>
Subject: Re: [allenai/interactron] Wrong label for objects in visualizations (Issue #4)

Hi, I also met the same issue, and the authors seem not update their codes. Can you send me your revised code?

**Comment by cazhang** [comment]
did you notice "stove_burner" appear twice in the tlvis_classes? would that be an issue?
